english/disadvantage_of_git

```diff
@@ -0,0 +1 @@
+Subproject commit 1018a704d5a7b8c6c202006be542f3feff2264a7
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.